azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java

```diff
@@ -121,122 +128,107 @@ private FlowContainer(final Path projectDirPath, final Props props)
       this.globalProps = new Props(null, globalPropsPath);
     }
 
-    // Setup DAO, a lot of it is redundant
-    final DataSource dataSource= new MySQLDataSource(this.azKabanProps,
-        new DBMetrics(new MetricsManager(new MetricRegistry())));
+    // execution dir
+    this.execDirPath = execDirPath;
 
-    final DatabaseOperator dbOperator = new DatabaseOperator(
-        new QueryRunner(dataSource));
-
-    this.executorLoader = setupDao(dbOperator);
+    this.executorLoader = SERVICE_PROVIDER.getInstance(JdbcExecutorLoader.class);
+    logger.info("executorLoader from guice :" + this.executorLoader);
 
     // project Loader
-    this.projectLoader = new JdbcProjectImpl(this.azKabanProps, dbOperator);
+    this.projectLoader = SERVICE_PROVIDER.getInstance(JdbcProjectImpl.class);
+    logger.info("projectLoader from guice : " + this.projectLoader);
 
-    // setup executor service, TODO : revisit
+    // setup executor service
     this.executorService = Executors.newSingleThreadExecutor();
 
-    this.jobLogChunkSize = this.azKabanProps.getString("job.log.chunk.size", "5MB");
-    this.jobLogNumFiles = this.azKabanProps.getInt("job.log.backup.index", 4);
-    this.validateProxyUser = this.azKabanProps.getBoolean("proxy.user.lock.down", false);
+    this.jobLogChunkSize = this.azKabanProps.getString("job.log.chunk.size",
+            DEFAULT_LOG_CHUNK_SIZE);
+    this.jobLogNumFiles = this.azKabanProps.getInt("job.log.backup.index", DEFAULT_LOG_NUM_FILES);
+    this.validateProxyUser = this.azKabanProps.getBoolean("proxy.user.lock.down",
+            Constants.DEFAULT_VALIDATE_PROXY_USER);
     this.jobTypeManager =
         new JobTypeManager(
             this.azKabanProps.getString(AzkabanExecutorServer.JOBTYPE_PLUGIN_DIR,
                 PluginManager.JOBTYPE_DEFAULTDIR),
```

Comment:
Nit: Can you please rename PluginManager. JOBTYPE_DEFAULTDIR to DEFAULT_JOBTYPE_DIR?

Reply:
This is an existing code. Will handle it in subsequent PRs.